src/frontend/static/css/custom.css

```diff
@@ -0,0 +1,13 @@
+
+.winnerResultStyle {
+  color:#00cc77;
+  font-weight:500;
+}
+
+.loserResultStyle {
+  color:red
```

Comment:
Maybe we can reuse bulma red https://bulma.io/documentation/helpers/color-helpers/

Reply:
Maybe, but I think it is clean to have a class that corresponds semantically to "here is how to format the losing result" instead of simply hardcoding a color class?

Reply:
More "red" itself is 100%, not matching with rest of the site.

```css
loserResultStyle {
  color: hsl(348, 100%, 61%);
}
```

Reply:
Same for green